hw/hw3-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Download jar file to ensure correct driver
+wget --quiet -c https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-8.0.21.tar.gz -O - | tar -zxvf - mysql-connector-java-8.0.21/mysql-connector-java-8.0.21.jar --strip-components=1
+
+# Would not work with -C option, so jankily moving to jar folder in pyspark
+mv mysql-connector-java-8.0.21.jar ../venv/lib/python3.8/site-packages/pyspark/jars/
```

Comment:
It looks like you were able to figure this out without the help of the post the other day, great job here

Reply:
Agreed!